l1-contracts/bootstrap.sh

```diff
@@ -43,7 +43,8 @@ function build {
     # Step 2: Build the the generated verifier contract with optimization.
     forge build $(find generated -name '*.sol') \
       --optimize \
-      --optimizer-runs 200
+      --optimizer-runs 1 \
```

Comment:
isnt this the only line you needed to change? Or are we fine with this tradeoff for all contracts?

Reply:
Just for the meantime while the byte code is too big, without using external library's